velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -407,6 +437,28 @@ std::shared_ptr<const ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
   return nullptr;
 }
 
+std::shared_ptr<const ParquetTypeWithId> ReaderBase::getElementTypeWithId(
```

Comment:
wonder do we actually need to have this as a new function? looks it is just calling into an initialization and doesn't do anything in addition, why not just calling the initialization directly